lte/gateway/Makefile

```diff
@@ -120,31 +117,6 @@ build_connection_tracker:
 build_envoy_controller: ## Build envoy controller
 	cd $(MAGMA_ROOT)/feg/gateway && $(MAKE) install_envoy_controller
 
-test_python: ## Run all Python-specific tests
-	sudo service magma@* stop
-	make -C $(MAGMA_ROOT)/lte/gateway/python test_all
-
-test_oai: ## Run all OAI-specific tests
```

Comment:
It seems this is needed by `coverage_oai`. Was this an oversight?

Reply:
Hm, `coverage_oai` is supposed to be removed in https://github.com/magma/magma/pull/14264/files#diff-41d3ad27a8ccc4fa615212321c1f0d4d49af2a4920598f9c1283ea1856c8820b but this could take some time for the review. I will take it out here and then remove `test_oai` separately later.

Reply:
Pushed

Reply:
I created a PR https://github.com/magma/magma/pull/14381 to be merged later, please have a look at that too then.